docs/source/API/core/parallel-dispatch/ParallelReduceTag.rst

```diff
@@ -0,0 +1,50 @@
+``ParallelReduceTag()``
+=======================
+
+.. role::cpp(code)
+    :language: cpp
+
+Header File: ``<Kokkos_ExecPolicy.hpp>``
+
+.. _text: ../parallel-dispatch/parallel_reduce.html
+
+.. |text| replace:: ``parallel_reduce``
+
+A tag used in team size calculation functions to indicate that the functor for which a team size is being requested is being used in a |text|_
```

Comment:
Might be easier/more readable to use a cross reference to parallel_reduce here

Reply:
done